Index.html

```diff
@@ -0,0 +1,441 @@
+          be concerned with whether methods are public, private, or protected, and
+          you do not have to implement interfaces. Variables, parameters, and
+          function return types are not explicitly typed.
+        </p>
+      </article>
+    </section>
+    <section class="main-section" id="Hello_world">
+      <header>Hello world</header>
+      <article>
+        To get started with writing JavaScript, open the Scratchpad and write your
+        first "Hello world" JavaScript code:
+        <code
+          >function greetMe(yourName) { alert("Hello " + yourName); }
+          greetMe("World");
+        </code>
+  
+        Select the code in the pad and hit Ctrl+R to watch it unfold in your
+        browser!
+      </article>
+    </section>
+    <section class="main-section" id="Variables">
+      <header>Variables</header>
+      <p>
+        You use variables as symbolic names for values in your application. The
+        names of variables, called identifiers, conform to certain rules.
+      </p>
+      <p>
+        A JavaScript identifier must start with a letter, underscore (_), or
+        dollar sign ($); subsequent characters can also be digits (0-9). Because
+        JavaScript is case sensitive, letters include the characters "A" through
+        "Z" (uppercase) and the characters "a" through "z" (lowercase).
+      </p>
+      <p>
+        You can use ISO 8859-1 or Unicode letters such as å and ü in identifiers.
+        You can also use the Unicode escape sequences as characters in
+        identifiers. Some examples of legal names are Number_hits, temp99, and
+        _name.
+      </p>
+    </section>
+    <section class="main-section" id="Declaring_variables">
+      <header>Declaring variables</header>
+      <article>
+        You can declare a variable in three ways:
+        <p>
+          With the keyword var. For example, <code>var x = 42.</code> This syntax
+          can be used to declare both local and global variables.
+        </p>
+        <p>
+          By simply assigning it a value. For example, <code>x = 42.</code> This
+          always declares a global variable. It generates a strict JavaScript
+          warning. You shouldn't use this variant.
+        </p>
+        <p>
+          With the keyword let. For example,<code> let y = 13.</code> This syntax
+          can be used to declare a block scope local variable. See Variable scope
+          below.
+        </p>
+      </article>
+    </section>
+    <section class="main-section" id="Variable_scope">
+      <header>Variable scope</header>
+      <article>
+        <p>
+          When you declare a variable outside of any function, it is called a
+          global variable, because it is available to any other code in the
+          current document. When you declare a variable within a function, it is
+          called a local variable, because it is available only within that
+          function.
+        </p>
+  
+        <p>
```

Comment:
Please, try to keep your codes clean, 😸 

blank lines are not good professionaly.

Reply:
- [Optional] Please do not forget about requesting the student to fix linters errors proper, just for now, as it is in your project